src/script/user/UserRepository.ts

```diff
@@ -586,7 +574,27 @@ export class UserRepository extends TypedEventEmitter<Events> {
     );
   }
 
-  private mapUserResponse(found: APIClientUser[], failed: QualifiedId[], dbUsers?: UserRecord[]): User[] {
+  // Replaces a deleted user name ("default") with the name from the local database.
+  private replaceDefaultUserName(apiUsers: APIClientUser[], dbUsers: UserRecord[]): UserRecord[] {
+    return apiUsers.map(user => {
+      if (user.name !== 'default') {
```

Comment:
Just curious, did we decide to use `deleted` prop instead of name?

Reply:
If we talk about deleted users then we should rely on the is isDeleted attribute of the User Entity instead of the name.

Reply:
You're right, good catch. I've just updated the check (and the function name)